implementation/index.html

```diff
@@ -0,0 +1,296 @@
+<!doctype html>
+<meta charset="utf-8">
+<script src="BeforeInstallPromptEvent.js">
+</script>
+<style>
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+#installprompt {
+  animation: fadein 2s;
+  width: 10cm;
+  padding: 1cm;
+}
+</style>
+<button onclick="notifyBeforeInstallPrompt(this)">
+  Simulate install BIP
+</button>
+<button onclick="showInstallPrompt(this)">
+  Simulate manual "Add to Home Screen"
+</button>
+<script>
+/*globals BeforeInstallPromptEvent, DOMException*/
+"use strict";
+//
+{
+  const assertion = "No arguments should throw";
+  try {
+    new BeforeInstallPromptEvent();
+    console.assert(false, assertion);
+  } catch (err) {
+    console.assert(true, assertion);
+  }
+} {
+  const assertion = "First argument stringifies";
```

Comment:
I don't think the argument will stringify anything... but it itself may be stringified